ProyectoIntegrAccion.json

```
@@ -0,0 +1,12 @@
+{
+  "name": "Proyecto IntegrAcción",
+  "type": "personal",
+  "bio": "un proyecto de cooperación entre la Universidad Católica San Pablo y ACNUR"
```

a comma is necessary / una coma es necesaria
```suggestion
  "bio": "un proyecto de cooperación entre la Universidad Católica San Pablo y ACNUR",
```